cloudinit/config/schema.py

```diff
@@ -91,7 +92,14 @@ def get_jsonschema_validator():
     # This allows #cloud-config to provide valid yaml "content: !!binary | ..."
 
     strict_metaschema = deepcopy(Draft4Validator.META_SCHEMA)
-    strict_metaschema['additionalProperties'] = False
+    strict_metaschema["additionalProperties"] = False
+
+    # This additional label allows us to specify a different name
+    # than the property key. This is especially useful when using a
+    # "patternProperties" regex.
```

Comment:
Nit: This change happens in the `get_jsonschema_validator()`, but the purpose for this change is for generating meaningful doc labels. It might be helpful to explain "why it's helpful when using a patternProperties regex"

perhaps:
```
    # "patternProperties" regex, otherwise the property label in the generated docs will be a regular expression.
```